Applications/SlicerApp/Testing/Python/FiducialLayoutSwitchBug1914.py

```diff
@@ -203,12 +219,14 @@ def run(self,enableScreenshots=0,screenshotScaleFactor=1):
       rasDiff = math.pow((seedRAS[0] - volumeRAS[0]),2) + math.pow((seedRAS[1] - volumeRAS[1]),2) + math.pow((seedRAS[2] - volumeRAS[2]),2)
       if rasDiff != 0.0:
         rasDiff = math.sqrt(rasDiff)
-      print('Checking the difference between fiducial RAS position',seedRAS,'and volume RAS as derived from the fiducial display position',volumeRAS,': ',rasDiff)
+      print('Checking the difference between fiducial RAS position',seedRAS,
+            'and volume RAS as derived from the fiducial display position',volumeRAS,': ',rasDiff)
       if rasDiff > self.maximumRASDifference:
-        self.delayDisplay("RAS coordinate difference is too large as well!\nExpected < %g but got %g" % (self.maximumRASDifference, rasDiff))
-        return False
+        slicer.util.delayDisplay("RAS coordinate difference is too large as well!\nExpected < %g but got %g" % (self.maximumRASDifference, rasDiff))
+        self.delayDisplay('Test failed!')
```

Comment:
Why do we still use self.delayDisplay?
Instead of moving an assert later down (that repeats the rasDiff>self.maximumRASDifference test) it would be simpler to raise an exception (the exception makes the exception description visible in the output and makes the test fail).

Reply:
I can fix the assert stuff. 

Regarding self.delayDisplay are you thinking everything should be using slicer.util.delayDisplay instead of also having a delayDisplay method in ScriptedLoadableModuleTest that then calls slicer.util.delayDisplay? Having the method in the test class helps understand what type of common methods would be use in the testing class.

Here in this test class it does seem weird to have usages of both slicer.util.delayDisplay and self.displayDisplay so I can make it consistent and just use self.delayDisplay.

Reply:
Having a separate delayDisplay in ScriptedLoadableModuleTest makes sense because it may allow changing the scale of the delay for all tests. For example, if a developer watches the test while running he wants the message to be displayed for a long enough so that the text can be read. However, if the test is run automatically they the delay can be set to a much shorter value.

delayDisplay in ScriptedLoadableModuleTest may call slicer.util.delayDisplay internally to reduce code duplication.

Reply:
Ok I've pushed updates to the assert stuff. Also test classes updated in this PR are using delayDisplay of ScriptedLoadableModuleTest instead of `slicer.util.delayDisplay`. Anything delayDisplay usage in ScriptedLoadableModuleLogic uses `slicer.util.delayDisplay`.